crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -187,6 +187,55 @@ type Epoch {
 	protocolConfigs: ProtocolConfigs
 }
 
+type Event {
+	id: ID!
+	sendingModuleId: MoveModuleId
+	eventType: MoveType
+	senders: [Address!]
+	timestamp: DateTime
+	json: String
+	bcs: Base64
+}
+
+type EventConnection {
+	"""
+	Information to aid in pagination.
+	"""
+	pageInfo: PageInfo!
+	"""
+	A list of edges.
+	"""
+	edges: [EventEdge!]!
+	"""
+	A list of nodes.
+	"""
+	nodes: [Event!]!
+}
+
+"""
+An edge in a connection.
+"""
+type EventEdge {
+	"""
+	The item at the end of the edge
+	"""
+	node: Event!
+	"""
+	A cursor for use in pagination
+	"""
+	cursor: String!
```

Comment:
This should be optional?

Reply:
man all these options and '!'s are gonna kill me

Reply:
Haha, I know, been through them too. :))

Reply:
actually it looks like the other edges all have cursor: String! and I personally think we should return a cursor on these edges anyways.. i think i'll keep it as is